.buildkite/e2e_on_pull_requests/check_if_we_need_to_ask.sh

```diff
@@ -17,6 +17,12 @@ GITHUB_API_TOKEN=$(aws secretsmanager get-secret-value \
   --secret-id builds/github_wecobot/e2e_pull_request_labels \
   | jq -r .SecretString)
 
+    curl -L \
+      -H 'Accept: application/vnd.github+json' \
+      -H "Authorization: Bearer $GITHUB_API_TOKEN" \
+      -H "X-GitHub-Api-Version: 2022-11-28" \
+      "https://api.github.com/repos/wellcomecollection/wellcomecollection.org/issues/$BUILDKITE_PULL_REQUEST/labels" | jq .
```

Comment:
What does this do?

Reply:
Looks like it might be an accidental copy/paste of the lines below it?